### PR TITLE
feat: Shop cart page added

### PR DESCRIPTION
**Pull request by Creator-singh**
cart_page

### CODE REVIEW

**Review by aniketsingh13 — APPROVED** [review]
look good to me

File: styles/cart.css [hunk 1]
```diff
@@ -0,0 +1,156 @@
...
+}
+
+.cart-cta{
+    margin: 2rem 0;
+}
+
+.remove{
+    background-color: var(--primary-clr);
+    border: 1px solid var(--primary-clr);
+    color: white;
```
Comment (prakashsakari) [inline styles/cart.css]: you can use variables instead of directly specifying the color

**Review by prakashsakari — COMMENTED** [review]
added a comment, rest looks good to me

File: styles/cart.css (hunk 1 above)
Comment (Creator-singh) [reply styles/cart.css]: okay will change

**Review by prakashsakari — APPROVED** [review]
Looks good to me

**Pull request merged**